oblt-cli/undeploy-my-kibana/action.yml

```diff
@@ -0,0 +1,90 @@
+name: 'oblt-cli/undeploy-my-kibana'
+description: 'Undeploy my kibana given the Pull Request'
+inputs:
+  pull-request:
+    description: 'The GitHub Pull Request ID'
+    default: ${{ github.event.pull_request.number }}
+  repository:
+    description: 'The GitHub repository'
+    default: ${{ github.repository }}
+  github-token:
+    description: 'The GitHub Personal Access Token.'
+    required: false
+  github-app-id:
+    description: 'The GitHub App ID to generate the ephemeral token.'
+    required: false
+  github-app-private-key:
+    description: 'The GitHub App Private Key to generate the ephemeral token.'
+    required: false
```

Comment:
I don't know whether keeping both approaches might be good... WDYT?

Reply:
I think the GitHub app credentials are only a temporary solution. I understand the benefit right now, but I don't think we want GitHub app credentials as GitHub secrets in the future, as they are very powerful.

Also, as we know, a new approach is on the horizon already.

Hence, we should not add it here and just pass the GH token from the output of the tibdex action.

Reply:
I think it's even more protected this implementation since it pretty much generates the token with the expected permissions. 
While if using the other approach implies the consumers will need to know what kind of permissions need be generated. 

I somehow feel this approach, even if temporary, will facilitate the consumers since they don't need to know much. WDYT?

Reply:
Good point.

On the other hand, there is only one consumer. https://github.com/search?q=org%3Aelastic+elastic%2Fapm-pipeline-library%2F.github%2Factions%2Fundeploy-my-kibana+%28path%3A**%2F*.yml+OR+path%3A**%2F*.yaml%29&type=code

And it's basically us maintaining it. (this may be true also for other oblt-actions, but I think this one is even more true for this one)

I'm also torn now.

The only thing I can think of is that we don't have to remove the inputs in a breaking change in the future if we don't add them in the first place.

Reply:
I agree about the `breaking change`. I initially didn't implement see https://github.com/elastic/oblt-actions/pull/140/commits/dcb19adddbeee723e1bd37f89090f32af6b2478b, but to a certain extent, I found it a bit complicated when I wrote the tests, see https://github.com/elastic/oblt-actions/pull/140/commits/d0db167c7d35e5ed6950917d88aeca93a734366a - it's much cleaner.

In any case, we can support both cases and then once we have the new GH ephemeral token, the change will be pretty much using the `github-token` input, so we won't need to care about the other two input parameters, we just won't use them